cypress/support/commands/googleLogin.ts

```diff
@@ -0,0 +1,38 @@
+import "@testing-library/cypress/add-commands"
+
+Cypress.Commands.add("loginByGoogleApi", () => {
+  cy.log("Logging in to Google")
+  cy.request({
+    method: "POST",
+    url: "https://www.googleapis.com/oauth2/v4/token",
+    body: {
+      grant_type: "refresh_token",
+      client_id: Cypress.env("googleClientId"),
+      client_secret: Cypress.env("googleClientSecret"),
+      refresh_token: Cypress.env("googleRefreshToken"),
```

Comment:
@RubySpeeders I _think_ that we need to add these variables to the GitHub repo settings in order to successfully authenticate. This would be the information you have in your `.env` file I believe.